OTLP -> Prom otel_scope metrics and labels updated to spec (#5246)
grafana/agent: docs/sources/flow/reference/components/otelcol.exporter.prometheus.md

metrics. Set the `include_scope_info` and `include_target_info` arguments to
`false`, respectively, to disable the custom metrics.

> `include_scope_labels` is used to enable including the `otel_scope_name` and
> `otel_scope_version` labels to every converted metric sample.

Reviewer: ```suggestion
When `include_scope_labels` is `true`  the `otel_scope_name` and
`otel_scope_version` labels are added to every converted metric sample.
```

I am guessing that you need to set this new arg to true?

Author: ya that sounds better
